Build error caused by duplicate fn definition

From build step on `master`: https://github.com/unicode-org/icu4x/runs/1024819152

```
error[E0201]: duplicate definitions with name `as_inversion_list`:
   --> components/uniset/src/uniset.rs:154:5
    |
73  | /     pub(crate) fn as_inversion_list(&self) -> &[u32] {
74  | |         &self.inv_list
75  | |     }
    | |_____- previous definition of `as_inversion_list` here
...
154 | /     pub(crate) fn as_inversion_list(&self) -> &[u32] {
155 | |         &self.inv_list
156 | |     }
    | |_____^ duplicate definition
```

--- a/components/uniset/src/uniset.rs
+++ b/components/uniset/src/uniset.rs
@@ -150,11 +150,6 @@ impl UnicodeSet {
         self.contains_query(query as u32).is_some()
     }
 
-    /// Returns the inversion list as a slice
-    pub(crate) fn as_inversion_list(&self) -> &[u32] {
-        &self.inv_list
-    }
-
     /// Checks to see if the range is in the UnicodeSet, returns a Result
     ///
     /// Runs a binary search in `O(log(n))` where `n` is the number of start and end points

